setup.py

```diff
@@ -31,79 +18,97 @@
     print(msg, file=sys.stderr)
     sys.exit(1)
 
-# Directory of the current file in the (hopefully) most reliable way
-# possible, according to krischer
-SETUP_DIRECTORY = os.path.dirname(os.path.abspath(inspect.getfile(
-    inspect.currentframe())))
+
+DESCRIPTION = 'A set of functions for processing Gem datalogger files.'
+LONG_DESCRIPTION = """
+gemlog is a python package that provides functions for processing the
+log files from the Gem infrasound datalogger. The Gem infrasound logger
+is a low-cost, lightweight, low-power instrument for recording infrasound
+in field campaigns.
+
+Source code: https://github.com/ajakef/gemlog
+"""
+
+DISTNAME = 'gemlog'
+LICENSE = 'GPL-3.0'
+AUTHOR = 'Jake Anderson'
+MAINTAINER_EMAIL = 'ajakef@gmail.com'
+URL = 'https://github.com/ajakef/gemlog'
+
+
+# TO-DO: replace this with versioneer
+version_dict = {}
+version_path = convert_path('gemlog/version.py')
+with open(version_path) as version_file:
+    exec(version_file.read(), version_dict)
+VERSION = version_dict['__version__']
+
 INSTALL_REQUIRES = [
     'obspy',
     'numpy>=1.15.0',
+    'pandas>=1.0.0',
```

Comment:
Pandas was a missing requirement (possibly introduced for the first time in #1).  Pandas 1.0.0 is [fairly new](https://pypi.org/project/pandas/#history) but if there isn't a good reason to support <1.0.0 then I'd say just set the min at 1.0.0 like this.